demo/start.sh

```diff
@@ -1,24 +1,29 @@
 #!/usr/bin/env bash
 
+# Disable unicode.
+LC_ALL=C
+LANG=C
```

Comment:
I've never heard of needing to do this before. Can you provide any references - what does this change, why is it a good thing to do, etc?

Reply:
this section is good for optimization,
in this script the Unicode is not necessary anywhere so is useful to disable the Unicode for performance optimization
## Some references
[https://tldp.org/LDP/abs/html/optimizations.html
](https://tldp.org/LDP/abs/html/optimizations.html)
[https://github.com/dylanaraps/pure-bash-bible#disable-unicode](https://github.com/dylanaraps/pure-bash-bible#disable-unicode)

Reply:
It seems like it only makes a difference if the script is doing repeated string matching. I'm not in favour of cluttering these scripts like this unless it actually makes any difference.